egs/chime5/s5b/run.sh

```diff
@@ -0,0 +1,280 @@
+fi
+
+if [ $stage -le 9 ]; then
+  # Starting basic training on MFCC features
+  steps/train_mono.sh --nj $nj --cmd "$train_cmd" \
+		      data/${train_set}_30kshort data/lang exp/mono
+fi
+
+if [ $stage -le 10 ]; then
+  steps/align_si.sh --nj $nj --cmd "$train_cmd" \
+		    data/${train_set} data/lang exp/mono exp/mono_ali
+
+  steps/train_deltas.sh --cmd "$train_cmd" \
+			2500 30000 data/${train_set} data/lang exp/mono_ali exp/tri1
+fi
+
+if [ $stage -le 11 ]; then
+  steps/align_si.sh --nj $nj --cmd "$train_cmd" \
+		    data/${train_set} data/lang exp/tri1 exp/tri1_ali
+
+  steps/train_lda_mllt.sh --cmd "$train_cmd" \
+			  4000 50000 data/${train_set} data/lang exp/tri1_ali exp/tri2
+fi
+
+if [ $stage -le 12 ]; then
+  utils/mkgraph.sh data/lang exp/tri2 exp/tri2/graph
+  for dset in ${test_sets}; do
+    steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+		    exp/tri2/graph data/${dset} exp/tri2/decode_${dset} &
+  done
+  wait
+fi
+
+#if [ $stage -le 13 ]; then
+#  steps/get_prons.sh --cmd "$train_cmd" data/train data/lang_nosp exp/tri2
+#  utils/dict_dir_add_pronprobs.sh --max-normalize true \
+#    data/local/dict_nosp exp/tri2/pron_counts_nowb.txt \
+#    exp/tri2/sil_counts_nowb.txt \
+#    exp/tri2/pron_bigram_counts_nowb.txt data/local/dict
+#fi
+#
+#if [ $stage -le 14 ]; then
+#  utils/prepare_lang.sh data/local/dict "<unk>" data/local/lang data/lang
+#  cp -rT data/lang data/lang_rescore
+#  cp data/lang_nosp/G.fst data/lang/
+#  cp data/lang_nosp_rescore/G.carpa data/lang_rescore/
+#
+#  utils/mkgraph.sh data/lang exp/tri2 exp/tri2/graph
+#
+#  for dset in dev test; do
+#    steps/decode.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+#      exp/tri2/graph data/${dset} exp/tri2/decode_${dset}
+#    steps/lmrescore_const_arpa.sh --cmd "$decode_cmd" data/lang data/lang_rescore \
+#       data/${dset} exp/tri2/decode_${dset} exp/tri2/decode_${dset}_rescore
+#  done
+#fi
+
+
+
+if [ $stage -le 14 ]; then
+  steps/align_si.sh --nj $nj --cmd "$train_cmd" \
+		    data/${train_set} data/lang exp/tri2 exp/tri2_ali
+
+  steps/train_sat.sh --cmd "$train_cmd" \
+		     5000 100000 data/${train_set} data/lang exp/tri2_ali exp/tri3
+fi
+
+if [ $stage -le 15 ]; then
+  utils/mkgraph.sh data/lang exp/tri3 exp/tri3/graph
+  for dset in ${test_sets}; do
+    steps/decode_fmllr.sh --nj $decode_nj --cmd "$decode_cmd"  --num-threads 4 \
+			  exp/tri3/graph data/${dset} exp/tri3/decode_${dset} &
+  done
+  wait
+fi
+
+if [ $stage -le 16 ]; then
+  # The following script cleans the data and produces cleaned data
+  steps/cleanup/clean_and_segment_data.sh --nj ${nj} --cmd "$train_cmd" \
+    --segmentation-opts "--min-segment-length 0.3 --min-new-segment-length 0.6" \
+    data/${train_set} data/lang exp/tri3 exp/tri3_cleaned data/${train_set}_cleaned
+fi
+
+if [ $stage -le 17 ]; then
+  rm -r data/train_worn_cleaned 2>/dev/null || true
+  utils/copy_data_dir.sh data/${train_set}_clean data/train_worn_cleaned
```

Comment:
@vimalmanohar The data/${train_set}_clean in this stage should be data/${train_set}_cleaned right?

Reply:
Yes, please fix it.